test/unit/controllers/declaration/PackageInformationControllerSpec.scala

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.controllers.declaration
+
+import controllers.declaration.PackageInformationController
+import controllers.util.{Add, Remove, SaveAndContinue}
+import forms.Choice.AllowedChoiceValues.SupplementaryDec
+import forms.declaration.PackageInformation
+import play.api.test.Helpers._
+import services.cache.ExportItem
+import unit.base.ControllerSpec
+import views.html.declaration.package_information
+
+class PackageInformationControllerSpec extends ControllerSpec {
+
+  trait SetUp {
+    val packageInformationPage = new package_information(mainTemplate)
+
+    val controller = new PackageInformationController(
+      mockAuthAction,
+      mockJourneyAction,
+      mockErrorHandler,
+      mockCustomsCacheService,
+      mockExportsCacheService,
+      stubMessagesControllerComponents(),
+      packageInformationPage
+    )(ec)
+
+    val itemId = "itemId"
+
+    authorizedUser()
+    withCaching(None)
+    withNewCaching(aCacheModel(withChoice(SupplementaryDec)))
+  }
+
+  "Package Information Controller" should {
+
+    "return 200 (OK)" when {
+
+      "display page method is invoked and cache is empty" in new SetUp {
+
+        val result = controller.displayPage(itemId)(getRequest())
+
+        status(result) must be(OK)
+      }
+
+      "display page method is invoked and cache contain some data" in new SetUp {
+
+        val itemWithPackageInformation =
+          ExportItem(itemId, packageInformation = List(PackageInformation(Some("12"), Some(10), Some("123"))))
```

Comment:
There's an `ExportsCacheItemBuilder` similar to the `ExportsCacheModelBuilder` if you think it helps aid readability

Reply:
You can do `aCacheItem(withPackageInformation(Some("12"), Some(10), Some("123")))`